packages/web-app-files/src/components/AppBar/ViewOptions.vue

```diff
@@ -1,5 +1,19 @@
 <template>
-  <div>
+  <div class="oc-flex oc-flex-middle">
+    <div class="oc-button-group oc-visible@s oc-mr-s">
+      <oc-button
+        :appearance="viewModeCurrent === 'resource-table-condensed' ? 'filled' : 'outline'"
```

Comment:
Done, good point. Took me way longer than necessary because I was wondering what `const` you were talking about exactly^^. Please take a look at https://github.com/owncloud/web/pull/7976/commits/e14e4645eba41592c27f19e10463cc465e2ef3e8 and let me know which of the two constant variants you prefer! (as in `ViewModeConstants.viewModeDefault` vis `ViewModeConstants.default`, e.g.)